### PR TITLE
separate the status cmd info into sub flag

### PR DESCRIPTION
**Pull request by xiangzepro**
## Description

What is the purpose of the change?
Extract sub info from status cmd by adding sub flag to simplify the info.

## Brief of your solution

Usage:
  xchain-cli status [flags]

Flags:
  -b, --branch   Get branch info
  -h, --help     help for status
  -l, --ledger   Get ledger info
  -p, --peers    Get peers info
  -u, --utxo     Get utxo info

### CODE REVIEW

File: core/cmd/cli/status.go [hunk 1]
```diff
@@ -58,6 +75,90 @@ func (s *StatusCommand) printXchainStatus(ctx context.Context) error {
 	return nil
 }
 
+func (s *StatusCommand) convertToFlag() pb.ViewOption {
```
Comment (yucaowang) [inline core/cmd/cli/status.go:78]: May I use multi flags in one command?

File: core/core/xchaincore.go [hunk 2]
```diff
@@ -1033,26 +1033,30 @@ func (xc *XChainCore) GetBlockChainStatus(in *pb.BCStatus) *pb.BCStatus {
...
+	if viewOption == pb.ViewOption_NONE || viewOption == pb.ViewOption_UTXOINFO {
+		utxoMeta := xc.Utxovm.GetMeta()
+		out.UtxoMeta = utxoMeta
+	}
 	ib, err := xc.Ledger.QueryBlock(meta.TipBlockid)
 	if err != nil {
 		out.Header.Error = HandlerLedgerError(err)
 		return out
 	}
 	out.Block = ib
```
Comment (yucaowang) [inline core/core/xchaincore.go:1048]: Do not return out.Block if option is not None.

File: core/cmd/cli/status.go (hunk 1 above)
Comment (xiangzepro) [reply core/cmd/cli/status.go:78]: Nope, only support one flag currently

File: core/core/xchaincore.go (hunk 2 above)
Comment (xiangzepro) [reply core/core/xchaincore.go]: Fixed, thanks

File: core/cmd/cli/status.go (hunk 1 above)
Comment (yucaowang) [reply core/cmd/cli/status.go:78]: OK

**Pull request closed without merging**